### PR TITLE
Remove logic to get labels from keys

### PR DESCRIPTION
**Pull request by dr0i**
One of several results of  #1846:

There is no more "@graph" and also no "subjectChain". Also every "id" has a "label".

### CODE REVIEW

File: web/app/views/TableRow.java [hunk 1]
```diff
@@ -34,44 +34,8 @@ public String process(JsonNode doc, String property, String param,
...
-				term = URLEncoder.encode(term, "UTF-8");
-			} catch (UnsupportedEncodingException e) {
-				Logger.error("Could not call encode '{}'", term, e);
-			}
-			String search = String.format("/resources/search?%s=%s", param, term);
-			JsonNode node = doc.get(property);
-			String label = labelForId(value, node, labels);
-			String result = labels.get().contains("numbering") ? label
-					: String.format(
-							"<a title=\"Nach weiteren Titeln suchen\" href=\"%s\">%s</a>",
```
Comment (fsteeg) [inline web/app/views/TableRow.java:63]: Don't we lose these links with this change? Also a few lines below, "Linked-Data-Quelle abrufen". Is this deployed somewhere to see the UI?

Comment (dr0i) [reply web/app/views/TableRow.java:63]: Now deployed to http://stage.lobid.org/resources/ .

Comment (fsteeg) [reply web/app/views/TableRow.java:63]: Alright, links are there (e.g. http://stage.lobid.org/resources/990067007460206441), wherever they are coming from.

**Review by fsteeg — APPROVED** [review]

**Comment by dr0i** [comment]
Will be deployed next Monday. Closed.

**Pull request merged**